collect/systemmetric.go

```diff
@@ -0,0 +1,78 @@
+package collect
```

Comment:
Right now this collector is only collecting allocated bytes metric since we haven't thought out a comprehensive list. So this file is just a WIP/placeholder until we do that